src/components/PassageMatches.tsx

```diff
@@ -38,7 +41,13 @@ const PassageMatches = ({ passages, onClick, activeIndex, pageColour = "textDark
       ) : (
         <div className="my-5" id="passage-matches">
           {passages.map((item, index: number) => (
-            <div key={item.text_block_id} data-analytics="document-passage-result" id={`passage-${index}`} className="mb-2">
+            <div
+              key={item.text_block_id}
+              data-analytics="document-passage-result"
+              id={`passage-${index}`}
+              className="mb-2"
+              onClick={() => posthog.capture("Passage matches click", { index })}
```

Comment:
To confirm for my benefit: the click event on the div below bubbles up to this event too?

Reply:
@PatFawbertMills Yup - any child (assume this is what you mean "below") of this will trigger this event.